src/Controls/tests/Core.UnitTests/MockPlatformServices.cs

```diff
@@ -84,6 +84,9 @@ public class MockApplication : Application
 		public MockApplication()
 		{
 		}
+
+		public void NotifyThemeChanged() =>
+			(this as IApplication).ThemeChanged();
```

Comment:
Do we really need the 'as'?

Reply:
If we want these changes we should do a PR to main and not lose the changes in this backport PR. 

I don't think we need to block the backport PR on this, but perhaps we get a PR to main with the fixes before we complete this backport PR?